src/common/options/global.yaml.in

```diff
@@ -2382,6 +2382,14 @@ options:
   default: true
   flags:
   - runtime
+- name: osd_calc_pg_upmaps_aggressively_fast
+  type: bool
+  level: advanced
+  desc: Prevent very long (>10 minutes) calculations in some extreme cases (applicable
+    only to aggressive mode)
+  default: true
```

Comment:
Is there any reason why we might want to set the default to `false`? IMO I would personally like to use this improvement off the bat, but we should consider best practices here. It looks like `osd_calc_pg_upmaps_aggressively` is set to `default: true` though, so this implementation aligns in that sense.